src/api/app/views/webui2/webui/projects/rebuild_times/show.html.haml

```diff
@@ -0,0 +1,35 @@
+:ruby
+  content_for(:content_for_head, javascript_include_tag('webui2/cm2/index-prjconf'))
+  @pagetitle = "Rebuild Time for Project #{@project}"
+  @metarobots = 'noindex'
+  p1 = @longestpaths[0].reverse
+  p2 = @longestpaths[1].reverse
+  p3 = @longestpaths[2].reverse
+
+.card
+  = render(partial: 'webui/project/tabs', locals: { project: @project })
+  .card-body
+    %h3
+      Rebuild Time: #{distance_of_time_in_words(Time.now, Time.now - @rebuildtime)} (#{format_seconds(@rebuildtime)})
+    %p
+      %img.rebuildgraph{ alt: 'The image could not be loaded. This should be a rebuild time graph.',
```

Comment:
The `alt` attribute should explain the content of the image. No why the image is not loaded.

Reply:
This is also used by the screen readers used by bling people

Reply:
Changed to `Rebuild time graph`.